Sources/NetworkProtection/ExtensionMessage/ExtensionRequest.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  ExtensionRequest.swift
+//
+//  Copyright © 2023 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+public enum DebugCommand: Codable {
+    case expireRegistrationKey
+    case sendTestNotification
+}
+
+public enum ExtensionRequest: Codable {
```

Comment:
The advantage of using this is two folded:

- Much simpler: adding a case is enough to be able to receive it in the extension, there's no need to add code to encode the data.
- Less likely to cause trouble: because the app and extension use this enum, adding a new case will present an error if the case handling is missing in either side.